spec/views/catalog/_form.html.haml_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe "catalog/_form.html.haml" do
+  before(:each) do
```

Comment:
`(:each)` is the default and thus you don't need to specify it (minor, so I won't let it hold up the merge)